test/src/api-action-creator-tests.js

```diff
@@ -236,6 +236,27 @@ describe('APIActionCreators', function () {
         r.abort();
     });
 
+		it('should call handleFailure on a network error', function (done) {
```

Comment:
This indenting seems a bit off 🤔

Reply:
Not sure how it went that badly wrong!
Updated 👍